pull_external.py

```diff
@@ -5,13 +5,15 @@
 import toml
 import subprocess
 import requests
+from requests.adapters import HTTPAdapter, Retry
 from typing import Dict, List, Set, Tuple, Pattern, Match
 from urllib.parse import urlparse
 from pathlib import Path
 
 CHECKOUT_DIR = "checkouts"
 GIT_CLONE_CMD = "git clone {{}} ./{}/{{}}/{{}}".format(CHECKOUT_DIR)
 GIT_CHECKOUT_CMD = "git checkout {}"
+GITHUB_API_RELEASES = "https://api.github.com/repos/spiffe/spire/releases"
 GITHUB_API_LATEST_RELEASE = "https://api.github.com/repos/spiffe/spire/releases/latest"
```

Comment:
nit: what about this to avoid the duplication of the URL?
```
GITHUB_API_RELEASES = "https://api.github.com/repos/spiffe/spire/releases"
GITHUB_API_LATEST_RELEASE = f'{GITHUB_API_RELEASES}/latest'
```